Source/GrasscutterTools/Resources/en-us/CustomCommands.txt

```diff
@@ -37,11 +37,11 @@ Reload config
 Re-login
 /kick
 Clear Backpack(ALL)
-/clear all
+/clear all lv90 r5 5*
```

Comment:
Is this the correct command? I don't see any parsing of these parameters from the code.

Reply:
Sorry, I just discovered the PR you referenced and it hasn't been merged yet.